docs/CHANGELOG.md

```diff
@@ -5,6 +5,10 @@ The format is based on [Keep a Changelog](https://keepachangelog.com/en/1.0.0/),
 and this project adheres to [Semantic Versioning](https://semver.org/spec/v2.0.0.html).
 
 ## [Unreleased]
+### Added
+- Image builder keeps track of the duration of each phase of image building in its metrics,
+  including: `fetching the source code`, `building image` and `pushing image` phases.
```

Comment:
It looks like there's a 4th metric in the screenshot, should that be included here?

Reply:
`preparing job type builder` is a bit puzzling, it takes only 21ms (and it should always take). It raises more questions than it explains, but I can put it there.